packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -75,10 +75,13 @@ export class TreeConfiguration<TSchema extends ImplicitFieldSchema = ImplicitFie
 	 * (meaning it does not even have any schema set at all).
 	 * If `initialTree` returns any actual node instances, they should be recreated each time `initialTree` runs.
 	 * This is because if the config is used a second time any nodes that were not recreated could error since nodes cannot be inserted into the tree multiple times.
+	 * @param enableSchemaValidation - If true, the tree will validate new content against its stored schema at insertion time
+	 * and throw an error if the new content doesn't match the expected schema.
 	 */
 	public constructor(
 		public readonly schema: TSchema,
 		public readonly initialTree: () => InsertableTreeFieldFromImplicitField<TSchema>,
+		public readonly enableSchemaValidation: boolean = false,
```

Comment:
TreeConfiguration feels iffy to me as a Class; do we want an ITreeConfiguration instead to be passed to schematize()?